Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -97,19 +96,16 @@ def compute_virtual_charge_on_spacecraft():
     on the boundary of the domain, compute the charge
     that WarpX thinks there should be on the spacecraft.
     """
-    # Get global array for the whole domain (across MPI ranks)
-    phi = PhiFPWrapper(include_ghosts=False)[:, :]
-    rho = RhoFPWrapper(include_ghosts=False)[:, :]
-
-    # Check that this codes correspond to the global size of the box
-    assert phi.shape == (nr + 1, nz + 1)
-    assert rho.shape == (nr + 1, nz + 1)
+    warpx = sim.extension.warpx
+    rho = warpx.multifab("rho_fp", level=0)
+    phi = warpx.multifab("phi_fp", level=0)
 
-    dr, dz = sim.extension.warpx.Geom(lev=0).data().CellSize()
+    dr, dz = warpx.Geom(lev=0).data().CellSize()
 
     # Compute integral of grad phi over surfaces of the domain
-    r = np.linspace(rmin, rmax, len(phi), endpoint=False) + (rmax - rmin) / (
-        2 * len(phi)
+    nr = phi.shape[0]
+    r = np.linspace(rmin, rmax, nr, endpoint=False) + (rmax - rmin) / (
+        2 * nr
     )  # shift of the r points because the derivaties are calculated in the middle
     face_z0 = (
         2 * np.pi * 1.0 / dz * ((phi[:, 0] - phi[:, 1]) * r).sum() * dr
```

Comment:
Looks correct/as before: the `:` is without ghost cells, `()` is with ghost cells.
https://pyamrex.readthedocs.io/en/latest/usage/compute.html